doc/nrf/app_dev/device_guides/hpf/hpf.rst

```diff
@@ -0,0 +1,24 @@
+.. _hpf_index:
+
+High-Performance Framework (HPF)
+################################
+
+.. note::
+
+   The HPF support in the |NCS| is currently experimental.
```

Comment:
It should be noted that HPF is supported only on L15.

Reply:
Why in the fault_hadling.rst file we are mentioning nRF54H20 then (line 14)? Shouldn't this be removed?

Reply:
Should be removed